src/main.js

```diff
@@ -350,6 +355,15 @@ Formsy.Form = React.createClass({
   // Validate the form by going through all child input components
   // and check their state
   validateForm: function () {
+    // SGG Hack: Formsy reeeeeallly really wants to validate, even on unmounting...
+    if (!this.props.doValidation) {
```

Comment:
However, even on unmounting components, Formsy decides to validate, so we also need to block the change here.